src/data/imprint/index.tsx

```diff
@@ -13,6 +13,6 @@ export interface Imprint {
 const imprint: TranslationCollection<Imprint> = data
 
 export const getImprint = (locale: Locale): Imprint =>
-  getTranslation(locale)(imprint)
+  filterByLocale(locale)(imprint)
```

Comment:
```suggestion
  extractSingleTranslation(locale)(imprint)
```
(for the current schema)